### ISSUE TITLE
darkread and mathjax conflict

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#5588]
1. Enable darkreader dark mode in CoCalc (in account preferences)
2. Open a Jupyter notebook or Markdown file that uses Mathjax rendering.
3. See this error
<img width="1221" alt="Screen Shot 2021-12-01 at 2 01 05 PM" src="https://user-images.githubusercontent.com/1276278/144321193-596a5f31-0b41-47b5-b128-4052f68cd21d.png">
:

WORKAROUNDS:
- Dismiss the error by clicking where it says "Dismiss this error".  Everything may work fine for the rest of the session, though maybe something involving math rendering will be broken. I have no idea.
- Don't use dark mode by unchecking the box next to "Dark mode: reduce eye strain by showing a dark background (via DARK READER)" in account preferences.
- Disable dark mode, open the file that uses Mathjax, then enable dark mode.  For the rest of your session everything should then work fine.

The problem happens only when dark mode has already been enabled and you attempt to enable Mathjax.

The actual bug fix may involve temporarily disabling dark mode while MathJax loads, then re-enabling it. That will be disturbing, but much less so than the browser crashing.  Also, of course, we should upgrade to the latest version of dark mode; maybe they fixed their bug.

**Comment by williamstein** [comment]
This is caused by https://github.com/darkreader/darkreader/pull/6957

For now we can fix this by instally darkready version 4.9.35 and absolute do NOT allow any later version.

Also, we should really convince the darkreader project to make  https://github.com/darkreader/darkreader/pull/6957 optional (on by default) so we can disable it, since it makes darkreader completely incompatible with MathJax.  I'll make fixing this a new ticket blocked on upstream.

**Comment by williamstein** [comment]
https://github.com/sagemathinc/cocalc/issues/5591
